.env.sample

```diff
@@ -21,6 +21,7 @@ EMAIL_SHIP=
 EXPERIMENTS_PAGINATE_BY=50
 HOSTNAME=localhost
 KINTO_BUCKET=main-workspace
+KINTO_BUCKET_MAIN=main
```

Comment:
Oh yeah so this really was how to do it eh, cool interesting, g/j making it work